FEATURES.md

```diff
@@ -0,0 +1,110 @@
+open62541 Supported Features
+============================
+
+
+## Supported Features
+
+| __**Service**__             |                                 |                      | Comment              |
+|:----------------------------|:--------------------------------|:--------------------:|:---------------------|
+| Discovery Service Set       |                                 |                      |                      |
+|                             | FindServers()                   |  :white_check_mark:  |                      |
+|                             | FindServersOnNetwork()          | :first_quarter_moon: | Branch: feature_mdns |
+|                             | GetEndpoints()                  |  :white_check_mark:  |                      |
+|                             | RegisterServer()                |  :white_check_mark:  |                      |
+|                             | RegisterServer2()               |     :full_moon:      | Branch: master       |
```

Comment:
It comes with OPC UA v1.03

> This Service allows a Server to register its DiscoveryUrls and capabilities with a Discovery Server.
It extends the registration information from RegisterServer with information necessary for
FindServersOnNetwork. This Service will be called by a Server or a separate configuration utility.
Clients will not use this Service.